runners/core-java/src/test/java/org/apache/beam/runners/core/triggers/TriggerStateMachinesTest.java

```diff
@@ -86,9 +86,6 @@ public void testStateMachineForAfterWatermark() {
         RunnerApi.Trigger.newBuilder()
             .setAfterEndOfWindow(RunnerApi.Trigger.AfterEndOfWindow.getDefaultInstance())
             .build();
-    AfterWatermarkStateMachine.FromEndOfWindow machine =
-        (AfterWatermarkStateMachine.FromEndOfWindow)
-            TriggerStateMachines.stateMachineForTrigger(trigger);
```

Comment:
Kind of weird that this exact logic is written twice. I don't know the origin, but I wonder if it was an automatic refactor last time!